deploy/backplane/lpsre/hypershift/management-cluster/10-lpsre-management-cluster-project.ClusterRole.yml

```
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-lpsre-management-cluster-project
+rules:
+- apiGroups:
+  - package-operator.run
+  resources: ['*']
+  verbs:
+  - get
+  - list
+  - watch
+  - update
```

```suggestion
  - patch
  - update
```
Is this missed? Typically would include along with `update`.